language/resource.language.fr_fr/strings.po

```diff
@@ -0,0 +1,2284 @@
+msgid "Series Finale"
+msgstr "Finale de la série"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31215"
+msgid "Season Finale"
+msgstr "Finale de la saison"
+
+msgctxt "#31216"
+msgid "Video decoder"
+msgstr "Décodeur vidéo"
+
+msgctxt "#31217"
+msgid "Pixel format"
+msgstr "Format des pixels"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31218"
+msgid "Mid-Season Finale"
+msgstr "Finale de mi-saison"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31219"
+msgid "Left header"
+msgstr "En-tête gauche"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31220"
+msgid "Purple"
+msgstr "Violet"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31221"
+msgid "Slate"
+msgstr "Ardoise"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31222"
+msgid "Coal"
+msgstr "Charbon"
+
+msgctxt "#31223"
+msgid "Gridlines"
+msgstr "Quadrillage"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31224"
+msgid "Pink"
+msgstr "Rose"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31225"
+msgid "Chalk"
+msgstr "Craie"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31226"
+msgid "Blush"
+msgstr "Rouge"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31227"
+msgid "Premiere"
+msgstr "Première"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31228"
+msgid "Aqua"
+msgstr "Aqua"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31229"
+msgid "Vaporwave"
+msgstr "Vaporwave"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31230"
+msgid "Calendar"
+msgstr "Calendrier"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31231"
+msgid "Customise highlight colours"
+msgstr "Personnaliser les couleurs de surbrillance"
+
+#: /1080i/Custom_1111_Dialog_ColourPicker.xml
+msgctxt "#31232"
+msgid "Gradient"
+msgstr "Dégradé"
+
+msgctxt "#31233"
+msgid "Seek to"
+msgstr "Aller à"
+
+msgctxt "#31234"
+msgid "Customise indicators"
+msgstr "Personnaliser les indicateurs"
+
+msgctxt "#31235"
+msgid "Indicators"
+msgstr "Indicateurs"
+
+msgctxt "#31236"
+msgid "Collection"
+msgstr "Collection"
+
+msgctxt "#31237"
+msgid "Progress"
+msgstr "Progrès"
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31238"
+msgid "Coral"
+msgstr "Corail"
+
+msgctxt "#31239"
+msgid "Footer"
+msgstr "Pied de page"
+
+#: /1080i/Custom_1115_Dialog_ColourPresets.xml
+msgctxt "#31240"
+msgid "Tropical"
+msgstr "Tropical"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31241"
+msgid "Colour scheme"
+msgstr "Schéma de couleurs"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31242"
+msgid "Blue Slate"
+msgstr "Blue Slate"
+
+#: /1080i/Includes_Items.xml
+msgctxt "#31243"
+msgid "Midnight Purple"
+msgstr "Midnight Purple"
+
+msgctxt "#31244"
+msgid "Now playing"
+msgstr "Lecture en cours"
+
+msgctxt "#31245"
+msgid "Autoscroll labels"
+msgstr "Défilement automatique des étiquettes"
+
+msgctxt "#31246"
+msgid "Autoscroll textboxes"
+msgstr "Défilement automatique des boîtes de texte"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31247"
+msgid "Colour highlight"
+msgstr "Surbrillance de la couleur"
+
+msgctxt "#31248"
+msgid "Building includes"
+msgstr "Construction des inclusions"
```

Comment:
Shouldn't translate the word "includes" as this is a technical term which refers to specific skin engine files (the `Includes.xml` files)

Reply:
Hi Jurial, It's fixed.

I may make some small adjustments as I see the translations in context, in use.